src/screens/ProfileScreen/elements/TextWithEmoji.tsx

```diff
@@ -4,12 +4,22 @@ import emojiRegex from 'emoji-regex';
 import {COLORS} from '../../../utils/theme';
 import {fonts} from '../../../utils/fonts';
 
-export function TextWithEmoji({text}: {text: string}) {
+export function TextWithEmoji({
+  text,
+  textStyle,
+  testId
+}: {
+  text: string;
+  textStyle: object;
+  testId: string;
+}) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props `textStyle` and `testId` have been added to the `TextWithEmoji` component. This is a good change as it allows for more customization of the component's style and easier testing with the `testId`. However, the prop types are not well defined. The `textStyle` prop is typed as `object`, which is too generic. It would be better to use `StyleProp<TextStyle>` from `react-native` to provide a more specific type. Similarly, `testId` should be optional since not all instances of this component will need a test ID.

```diff
-  textStyle: object;
-  testId: string;
+  textStyle?: StyleProp<TextStyle>;
+  testId?: string;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->